packages/e2e/test/web-extension/dapp-sdk/utils.ts

```diff
@@ -1,20 +1,40 @@
+/* eslint-disable promise/no-nesting */
 /* eslint-disable @typescript-eslint/no-explicit-any */
+import { Logger } from '@cardano-sdk/util-dev';
 import { UnwitnessedTx } from '@cardano-sdk/tx-construction';
 import type { ObservableWallet } from '@cardano-sdk/wallet';
 
 export const inspectAndSignTx = async ({
   connectedWallet,
   builtTx,
-  textElement
+  textElement,
+  logger
 }: {
   connectedWallet: ObservableWallet;
   builtTx: UnwitnessedTx;
   textElement: any;
+  logger: Logger;
 }) => {
-  const txDetails = await builtTx.inspect();
-  textElement.textContent = `Built: ${txDetails.hash}`;
-  const signedTx = await builtTx.sign();
-  textElement.textContent = `Signed: ${signedTx.tx.id}`;
-  await connectedWallet.submitTx(signedTx);
-  textElement.textContent = `Submitted: ${signedTx.tx.id}`;
+  builtTx
+    .inspect()
+    .then((txDetails) => {
```

Comment:
The original `await` syntax is preferable. Could you please revert this change (try-catch to handle errors)?

Reply:
Yea, not too attached to this. Will revert.

Reply:
This is reverted.